inductiva/simulators/schism.py

```diff
@@ -0,0 +1,41 @@
+"""SCHISM module of the API."""
+from typing import Optional
+
+from inductiva import types, tasks, simulators
+
+
+@simulators.simulator.mpi_enabled
+class SCHISM(simulators.Simulator):
+    """Class to invoke a generic SCHISM simulation on the API."""
+
+    def __init__(self):
+        super().__init__()
+        self.api_method_name = "schism.schism.run_simulation"
+
+    def run(self,
+            input_dir: types.Path,
+            num_scribes: int = 1,
```

Comment:
Does this work if n_vcpus is 1?

Reply:
I think it will depend on the simulation